Packs/SymantecEmailSecurity/Integrations/SymantecEmailSecurity/SymantecEmailSecurity.py

```diff
@@ -0,0 +1,1888 @@
+            "Status",
+            "Description",
+            "Email Direction",
+            "Remediation Action",
+            "Expiry Date",
+        ],
+        removeNull=True,
+    )
+
+    return CommandResults(
+        outputs_prefix=f"{INTEGRATION_PREFIX}.IOC",
+        outputs_key_field="iocBlackListId",
+        outputs=raw_response,
+        readable_output=readable_output,
+        raw_response=raw_response,
+    )
+
+
+@logger
+def action_ioc_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """Add, update, delete and renew multiple IOCs.
+
+    Args:
+        client (Client): A session to run HTTP requests to the API.
+        args (dict[str, Any]): Arguments passed down by the CLI to configure the request.
+
+    Returns:
+        CommandResults: Outputs of the command that represent an entry in the warroom.
+    """
+    domain = args.get("domain", "global")
+    action = "IOC" if args["action"] == "upload_ioc_json" else args["action"].upper()
+
+    api_row_action = None
+
+    if action in APIListAction.__members__:
+        api_list_action = APIListAction[action]
+    else:  # action in APIRowActionChoices
+        api_list_action = APIListAction.IOC
+        api_row_action = APIRowActionChoices[action]
+
+    iocs: list[IOC] = []
+
+    if entry_id := args.get("entry_id"):  # Build IOCs from an entry file.
+        if api_row_action:
+            raise DemistoException("The field `entry_id` is only compatible with `action=merge/replace/ioc`.")
+
+        file_entry = demisto.getFilePath(entry_id)
+
+        with open(file_entry["path"], "rb") as handler:
+            content = handler.read()
+
+        iocs = [IOC.from_dict(ioc, api_list_action) for ioc in json.loads(content)]
+    else:  # Build a single IOC from the given params.
+        if action == APIListAction.IOC:
+            raise DemistoException("`action=ioc` is only compatible with `entry_id`.")
+
+        if email_direction := args.get("email_direction"):
+            email_direction = EmailDirectionChoices[email_direction.upper()].value
+
+        if remediation_action := args.get("remediation_action"):
+            remediation_action = RemediationActionChoices[remediation_action.replace(" ", "_").upper()].value
+
+        iocs.append(
+            IOC(
+                IocBlacklistId=args.get("ioc_id"),
+                APIRowAction=api_row_action,
+                IocType=args.get("ioc_type"),
+                IocValue=args.get("ioc_value"),
+                Description=args.get("description"),
+                EmailDirection=email_direction,
+                RemediationAction=remediation_action,
+                api_list_action=api_list_action,
+            )
+        )
+
+    raw_response = client.action_ioc(
+        iocs=iocs,
+        domain=domain,
+        api_list_action=api_list_action,
+    )
+
+    readable_output = "## The following IOC(s) failed:" if raw_response else "## All IOC(s) were uploaded successfully."
```

Comment:
Needed some thing generic, it would look a bit weird with "were upload_ioc_json successfully" in the case of uploading a file. 
Got another suggestion?